spec/rrd/base_spec.rb

```diff
@@ -6,124 +6,124 @@
     @rrd = RRD::Base.new(RRD_FILE)
   end
 
-  it "should create a rrd file using dsl" do
-    File.should_not be_file(RRD_FILE)
+  it "create a rrd file using dsl" do
```

Comment:
The spec text should either keep the "should" or use the present simple, i.e. "creates an rrd file using dsl"

Reply:
Thank you for your feedback.

I have addressed the the input in the third commit and cleaned up two other things I've found unclean.